src/pages/home-pages/home-pages.pages.jsx

```diff
@@ -1,100 +1,216 @@
-          By unlocking the capabilities of our platforms and developing open standards and APIs,
-          we can create an environment for digital innovation to thrive.
-        </p>
+      <p>
+        At Hackney Council we believe in the potential of digital technologies
+        to rapidly improve public sector services. By unlocking the capabilities
+        of our platforms and developing open standards and APIs, we can create
+        an environment for digital innovation to thrive.
+      </p>
 
-        <p> That means more digital services designed around the needs of our professionals and general public users.
-           Services that offer simple and speedy access to the information and data that matters,
-           whenever and wherever it’s needed.
-           Cost effective, creative and sustainable solutions that accelerate the digital transformation of Hackney Council.
-        </p>
+      <p>
+        {" "}
+        That means more digital services designed around the needs of our
+        professionals and general public users. Services that offer simple and
+        speedy access to the information and data that matters, whenever and
+        wherever it’s needed. Cost effective, creative and sustainable solutions
+        that accelerate the digital transformation of Hackney Council.
+      </p>
 
-        <p>
-        Through co-creation and collaboration, we can quickly overcome the challenges,
-        bringing the benefits of digital innovation to more people across the public sector.
-        </p>
+      <p>
+        Through co-creation and collaboration, we can quickly overcome the
+        challenges, bringing the benefits of digital innovation to more people
+        across the public sector.
+      </p>
 
-        <h2>Vision</h2>
-        <div>
+      <h2>Vision</h2>
+      <div>
         <ul>
           <li> To have a source of truth for all APIs at HackIT </li>
-          <li> To prevent duplication between APIs, hence allows all APIs to be consistent and reusable </li>
-          <li> To integrate with Swagger to provide swagger definition within our Developer Hub </li>
-          <li> To provide detailed information for each of the APIs we have at HackIT </li>
+          <li>
+            {" "}
+            To prevent duplication between APIs, hence allows all APIs to be
+            consistent and reusable{" "}
+          </li>
+          <li>
+            {" "}
+            To integrate with Swagger to provide swagger definition within our
+            Developer Hub{" "}
+          </li>
+          <li>
+            {" "}
+            To provide detailed information for each of the APIs we have at
+            HackIT{" "}
+          </li>
         </ul>
-        </div>
+      </div>
     </div>
   );
 };
 
 export const HomePage2 = () => {
   return (
     <div id="need-of-dev-hub" className="main-container">
-      <h2>The need of a Developer Hub</h2><br/>
+      <h2>The need of a Developer Hub</h2>
```

Comment:
Please revert these changes, you removed the accordion for the dev hub user needs